lib/elastomer/client/bulk.rb

```diff
@@ -43,6 +43,48 @@ def bulk( body = nil, params = nil )
       end
     end
 
+    # Stream bulk actions from an Enumerator.
+    #
+    # Examples
+    #
+    #   ops = [
+    #     Bulk::index(document1, :_type => 'default-type', :_id => 1),
+    #     Bulk::index(document2, params2),
+    #     Bulk::delete(params3),
+    #   ]
```

Comment:
That's a pretty clever use of `send` to reuse the existing methods! I'll implement that now.